From 3adafee3c3dbe84f00dac790ce9bf5184d6ed2c9 Mon Sep 17 00:00:00 2001
From: Yadu MG <quic_ymg@quicinc.com>
Date: Fri, 20 Dec 2024 19:17:56 +0530
Subject: [PATCH] [DNM] QcomModulePkg: Disable Keymaster and VB Loading for Seca

Keymaster loading is not yet implemented for SA8797, disable
the feature temporarily along with VB.

Change-Id: Ie4f2f6751ab3e4b5d551e5f498843752f4bbd2b7
Signed-off-by: Yadu MG <quic_ymg@quicinc.com>
---
 makefile | 2 --
 1 file changed, 2 deletions(-)

diff --git a/makefile b/makefile
index 6ad1f6bbc3..03abdae4ea 100644
--- a/makefile
+++ b/makefile
@@ -346,8 +346,6 @@ ABL_FV_IMG: $(EDK_TOOLS_PATH_MARK_FILE)
 	-D ABL_OUT_DIR=$(ABL_OUT) \
 	-D VERIFIED_BOOT_LE=$(VERIFIED_BOOT_LE) \
 	-D FORCE_NO_PIE=$(FORCE_NO_PIE) \
-	-D VERIFIED_BOOT_ENABLED=$(VERIFIED_BOOT_ENABLED) \
-	-D LOAD_KM_AND_SET_ROT=$(LOAD_KM_AND_SET_ROT) \
 	-D PVM_SKIP_DTBO=$(PVM_SKIP_DTBO) \
 	-D EARLY_ETH_ENABLED=$(EARLY_ETH_ENABLED) \
 	-D ENABLE_SAIL_FLASHING=$(ENABLE_SAIL_FLASHING) \
-- 
2.43.0

